pom.xml

```diff
@@ -119,6 +119,9 @@
     <!-- The IIIF base URL (including service prefix) -->
     <iiif.base.url>https://iiif.library.ucla.edu/iiif/2</iiif.base.url>
 
+    <!-- The URL Fester is available at -->
+    <fester.url>https://iiif.library.ucla.edu</fester.url>
```

Comment:
Shouldn't the default value be localhost for dev/testing? We'd override on deployed versions?

Reply:
My inclination was to use prod like `iiif.base.url`, since `fester.url` isn't dereferenced anywhere anyway, but this makes sense. Will fix.